.pipelines/templates/.builder-release-template.yaml

```diff
@@ -43,6 +43,7 @@ steps:
       if [[ "${IMG_SKU}" == *"minimal"* ]]; then SKU_NAME="${SKU_NAME}minimal"; fi && \
       if [[ "${ENABLE_TRUSTED_LAUNCH}" == "True" ]]; then SKU_NAME="${SKU_NAME}TL"; fi && \
       if [[ ${OS_SKU} != "CBLMariner" && "${CONTAINER_RUNTIME}" == "containerd" ]]; then SKU_NAME="${SKU_NAME}containerd"; fi && \
+      if [[ ${OS_VERSION} == "V2" && ${OS_SKU} == "CBLMariner" && ${ENABLE_CGROUPV2,,} == "true" ]]; then SKU_NAME="${SKU_NAME}cgroupv2"; fi && \
```

Comment:
can we simplify this to just `${ENABLE_CGROUPV2,,} == "true"`?

Reply:
I'm also a bit confused since it seems like these would produce new SKUs?

Reply:
Good catch. I will remove this line. It was a leftover.

Reply:
after simplification this seems fine